### ISSUE TITLE
Change in RTF script formatting in 7.2.1.9

### ISSUE THREAD

**Issue by MCIR** [christillman/encounterpro_os#66]
A new issue I identified with version 7.2.1.9 is, the formatting has changed compared to previous versions I had configured. Is this an unintended result of working on prior meds or part of an ongoing effort to enhance the RTF script format? The new format is difficult to read. Titles are not clear, making it harder to distinguish sections of the patient summary note.

**Comment by christillman** [comment]
Hi Ciru,

I searched through 7.2.1.9 changes, but couldn't find any changes relating to display script formatting. It would be great to run 7.2.1.8 against the same database, record some screenshots, and run 7.2.1.9 again on the same patient without changing scripts, and record screenshots again, so I can see the differences clearly.

**Comment by MCIR** [comment]
Hi Chris

 

Same patient  Neria, Nathan same appointment

 

Same RTF Script. Id no 1001791

 

Version 7.2.1.8

 



 



 



 



 



 

Version 7.2.1.9

 

Same patient  Neria, Nathan same appointment

 

Same RTF Script. Id no 1001791

**Comment by christillman** [comment]
Hi Ciru, can you copy to me directly? The pictures didn't come through
GitHub.

**Comment by MCIR** [comment]
A brief recap on changes made to progress notes in 7.2.2.0. 
The new standard format:

1. **For the SOAP End Note section**: Add a bold title and hard left to the text below the title. This has been successful. See below

![image](https://github.com/christillman/encounterpro_os/assets/40364841/faa79782-8f8f-4590-b21d-bd281ffbf597)

3. **For Treatment Types**:  Treatment type titles should not print if no text/results have been added to the treatment types selected. 
![image](https://github.com/christillman/encounterpro_os/assets/40364841/932ab747-5c5f-4cde-b012-d68ada0b1022)

The changes are working as intended. 

A related issue that I should have raised at the time I raised the previous issues above is that, treatment-type titles (observation titles) are showing below the main treatment-type title leading to duplication. They are displayed at the beginning of the paragraph. 

For example, in the screenshot below, if the user wanted to add documentation to the 'History of Present Illness' section, and selected the 'ENT HPI' observation template, the template's title 'ENT HPI' precedes the text notes, below the main section title 'History of Present Illness'. 

Similarly, if the user wants to document under the 'Review of Systems' section and selects an appropriate template under this section, the template's title 'Review of Systems' precedes the text notes, below the main section title 'Review of Systems'. See below:

![image](https://github.com/christillman/encounterpro_os/assets/40364841/c6202c41-e943-4155-924b-7f099b1d7acd)

Is there a way to configure selected observation's titles to be hidden from the progress note but still allow them to be included when needed? 

I tried making changes to display styles OBO|RRR|CBN , ORO|RRR|CRN etc but it hasnt been successful because changing observation styles also changes the way the observation's text prints. 

In the second screenshot below, you'll see the observation titles I am referring to. These titles appear below the main observation title at the beginning of the paragraph and will usually share the same the same font as the rest of the text results. In this screenshot, I have bolded them for easier reference.  

![image](https://github.com/christillman/encounterpro_os/assets/40364841/29221772-492f-42ec-aa9d-6be51bc7e6bd)

**Comment by MCIR** [comment]
From: Chris Tillman <[toff.tillman@gmail.com](mailto:toff.tillman@gmail.com)> 
Sent: Saturday, 6 July 2024 06:45
To: [rcirur@gmail.com](mailto:rcirur@gmail.com)
Subject: Re: FW: [christillman/encounterpro_os] Change in RTF script formatting in 7.2.1.9 (Issue #66)-in case github images didnt come through

Hi Ciru,

When I look at the HPI for Xwnpx, it has two sections Free Text and Narrative.
![image](https://github.com/christillman/encounterpro_os/assets/40364841/beff1f79-9819-48af-9c8c-4710bb5b1405)


In this case I guess the subsection titles should not be removed. Would a good rule be to remove the subsection title only if there is one subsection? That would be like your example below where the subsection title ENT HPI is the only subsection under History of Present Illness .. so ENT HPI should be suppressed right? and the text should come to the left side.

**Comment by MCIR** [comment]
From: rcirur@gmail.com <rcirur@gmail.com> 
Sent: Saturday, 6 July 2024 18:31
To: 'Chris Tillman' <toff.tillman@gmail.com>
Subject: RE: FW: [christillman/encounterpro_os] Change in RTF script formatting in 7.2.1.9 (Issue #66)-in case github images didnt come through

In this case, I guess the subsection titles should not be removed?

No. It's the opposite. Subsection titles like ‘Narrative HPI’ and ‘Free Text HPI’ should be suppressed so they don't show in our progress note. 

These subsection titles reflect observation template names used for data entry e.g. ("Narrative HPI" and "Free Text HPI"). These template titles become redundant in the final document. 

In the example you are referring to, physicians selected templates with titles ‘Narrative HPI’ and ‘Free Text HPI’ to document under the ‘History of Present Illness’ section. 

A brief background:

A patient’s clinic summary note includes main sections such as ‘Chief Complaint’, ‘History of Present Illness’, ‘Allergies’, ‘Current Medications’, ‘Past Medical History’, ‘Review of Systems’, ‘Physical Exam’, ‘Investigations’, etc. The physician needs to select a relevant observation template to document under each section. For example, to document under the Review of Systems section, the physician needs to select a relevant ‘Review of Systems’ observation template. All templates under review of systems section have titles such as ‘ENT Review of Systems’ or ‘General Review of Systems’ or ‘Comprehensive Review of Systems’ and so on. 

Currently, in our progress note, any selected observation template’s title is showing as a subsection title under the main clinic progress summary note sections: ‘Chief Complaint’, ‘History of Present Illness’, ‘Past Medical History’, ‘Review of Systems’, ‘Physical Exam’, ‘Investigations’ making them redundant, hence the request to suppress them, so that we remain with the main section titles.  

An example can be seen below. 

The physician selected two different templates to document under the  ‘Review of Systems’ section. The first template was titled ‘Cardiovascular’ and the second one was titled ‘Review of Systems. 

In the progress note, the template titles ‘Cardiovascular’ and ‘Review of Systems’ appear in blue and precede the physician’s notes. 

I have bolded the titles to be suppressed,  ‘Cardiovascular’ and ‘Review of Systems’, for easier identification.  

 
See below, screenshots of the observation templates selected to document a ‘Review of Systems’. 

The first observation template ‘Cardiovascular’ is part of a larger set known as ‘Comprehensive Review of Systems’. See below. 
 


When you select the ‘Cardiovascular’ sub-section, it opens the cardiovascular template. 
 

On the progress note it prints as seen below. The observation template title prints below the main section heading ‘Review of Systems’.
 


The second template titled ‘ENT Review of Systems’ was also selected to document a ‘Review of Systems’ as seen below. 
 

On the progress note it prints in the format seen below, with the subsection title ‘Review of Systems’  

 
Since I selected two different templates to document a review of systems, results from both templates will appear below the main ‘Review of Systems’ title as seen below. 

 


Would a good rule be to remove the subsection title only if there is one subsection? 
In the progress note, I can say a good rule would be to remove all subsection titles. I am inclined to say all observation template titles. 

That would be like your example below where the subsection title ENT HPI is the only subsection under History of Present Illness .. so ENT HPI should be suppressed right? and the text should come to the left side.
Yes.
1.	The title ‘ENT HPI’ should be suppressed because it is the title of a template. 
2.	The text should come to the left side.

In the example below, the ‘Review of Systems’ subsection title in blue, located below the bolded ‘Review of Systems’ title should be suppressed and the text ‘Complains of dry nostrils.....’ should move to the left. 
 +

In the example below, the subsection titles are 
1.	‘Nose, Mouth and Throat’ right before ‘Nasal Mucosa’
and
2.	Nose, Mouth and Throat right before ‘Posterior Pharynx’


 

The template the physician selected to document is as seen below. The template contains sub-templates, each with its title.  See below:

The physician selected the ‘ENT Physical Examination’ observation. 
 

It is a template containing multiple sub-templates. From the main template the physician selected the  ‘Nose, Mouth and Throat’ section. 

 

Nose, Mouth and Throat is a sub-template of ‘ENT Physical Examination’ template. It also contains sub-templates as seen below

 

The physician documented under the ‘Nasal Mucosa’ template and ‘Posterior Pharynx’ Template. See below. 

 

Physician documented in the free text section of the ‘Nasal mucosa’ template. 

 

Below, the free text section of the ‘Nasal mucosa’ template

 


Another example, 


In the example below, both ‘Free text HPI and ‘Narrative HPI’ subsection titles should be suppressed.

**Comment by MCIR** [comment]
From: Chris Tillman <[toff.tillman@gmail.com](mailto:toff.tillman@gmail.com)> 
Sent: Friday, 5 July 2024 21:36
To: [rcirur@gmail.com](mailto:rcirur@gmail.com)
Subject: Re: FW: [christillman/encounterpro_os] Change in RTF script formatting in 7.2.1.9 (Issue #66)-in case github images didnt come through
 
Thanks! When you sent the previous email, there was a little improvement. Github at least included links to the images I could click on rather than just blank space. It's still helpful to have the images inline, so thanks for copying me directly also.
 
I had noticed the duplicated headings, and I believe you had mentioned it before also. I'd prefer to track it as a separate issue, so we could close off this one #66 if the changes addressed what it had previously. I created a new issue #70 for the duplicate header problem.
